src/visualizer.ts

```diff
@@ -2,13 +2,17 @@ import { render, h } from 'preact';
 import { UIRouter, UIRouterPlugin } from '@uirouter/core';
 import { StateSelector } from './selector/StateSelector';
 import { StateVisualizer } from './statevis/StateVisualizer';
-import { StateTree } from './statevis/tree/StateTree';
+import { NodeOptions, StateTree } from './statevis/tree/StateTree';
 import { TransitionVisualizer } from './transition/TransitionVisualizer';
 
 const visualizer = (router: UIRouter) => new Visualizer(router, {});
 
+export interface StateVisualizerOptions {
+  nodeOptions?: NodeOptions;
+}
+
 export interface Options {
-  state?: boolean;
+  state?: boolean | StateVisualizerOptions;
```

Comment:
I understand why you overloaded this property, but I'd prefer not to.  I also wish I had named these properties differently 😢  Can we move the state visualizer options to a separate property like `options.stateVisualizer`?